src/main/scala/is/hail/utils/TempDir.scala

```diff
@@ -38,7 +38,7 @@ object TempDir {
       createTempDir("/tmp", hConf))
 }
 
-class TempDir(localTempDir: String, tempDir: String) {
+case class TempDir(localTempDir: String, tempDir: String) {
```

Comment:
What is the reason for this change?

Reply:
Needed to access the `tempDir` argument for the test suite. Changed it to `val localTempDir...`